style.css

```diff
@@ -0,0 +1,76 @@
+* {
+  font-family: "Poppins", sans-serif;
+}
+
+nav {
+  display: flex;
+  margin: 20px;
+}
+
+.hamburger {
+  float: right;
+  margin-left: 60%;
+}
+
+body {
+  background-image: url("assets/images/shapesbg.svg");
+  background-repeat: no-repeat;
+}
+
+section {
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+  width: 100%;
+  height: 100%;
+}
+
+.header {
+  font-size: 44px;
+  font-weight: bold;
+  font-style: normal;
+  line-height: 52px;
+  display: flex;
+  align-items: center;
+  color: #172b4d;
+}
+
+.lets-connect {
+  position: static;
+  width: 117px;
+  height: 24px;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 24px;
+  color: #7f8cff;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
```

Comment:
- [ ] Kindly, resize the width of this element, I think you don't need to specify any fix width to it since it is breaking its design, please have a look to the image above ⏫  let's go and fix that 😄